### ISSUE TITLE
Prod preview - Unable to start pipelines

### ISSUE THREAD

**Issue by jyasveer** [openshiftio/openshift.io#2298]
I am unable to start pipelines in prod preview. And I am BLOCKED by this.

**Comment by joshuawilson** [comment]
@alexeykazakov could this have to with moving from free-int to free-stg?

**Comment by ldimaggi** [comment]
This looks OK now - is anyone still seeing this problem?
![screenshot from 2018-02-16 17-00-47](https://user-images.githubusercontent.com/642621/36331328-6346cdb4-133b-11e8-9f00-d05c141f029c.png)

**Comment by joshuawilson** [comment]
They manually scaled down my jenkins and back up. That brought Jenkins to life and I could create pipelines again. 

@jyasveer how long have you been unable to build / create pipelines?

**Comment by mmclanerh** [comment]
@jyasveer any update here? is this still an issue for you?

**Comment by sara-02** [comment]
@ldimaggi I am still seeing is error. 
![screenshot from 2018-02-23 15-48-52](https://user-images.githubusercontent.com/7047079/36589384-1388c1e4-18b1-11e8-99c3-0265288191b5.png)

**Comment by sara-02** [comment]
The screenshot is for a build that was created last week, but today's build is also not getting triggered.

**Comment by sara-02** [comment]
Jenkins is still not up for me. 
![screenshot from 2018-02-23 15-57-45](https://user-images.githubusercontent.com/7047079/36589837-b919dc5a-18b2-11e8-9a58-80ab2f4b3d0c.png)

**Comment by joshuawilson** [comment]
looks like a duplicate of #2341

**Comment by ldimaggi** [comment]
I was able to run a couple of build pipelines late on February 22, but Jenkins is a black box on prod-preview as we cannot login to its console due to this bug:  https://github.com/openshiftio/openshift.io/issues/2317

**Comment by mmclanerh** [comment]
@ldimaggi console should be fixed now.. you can also access openshift by command line.

**Comment by ldimaggi** [comment]
Verified - closing - thanks!
